Feature: OpenWRT service widget (#2782)
gethomepage/homepage: docs/widgets/services/openwrt.md

---
title: OpenWRT
description: OpenWRT widget configuration
---

Learn more about [OpenWRT](https://openwrt.org/).

Provides information from OpenWRT

```yaml
widget:
  type: openwrt
  url: http://host.or.ip
  username: homepage
  password: pass

>   interfaceName: eth0

Reviewer: ```suggestion
  interfaceName: eth0 # optional
```

```

## Interface

> Adding an interfaceName (e.g eth0) will display information for that particular device, otherwise it will display general system info.

Reviewer: ```suggestion
Setting `interfaceName` (e.g. eth0) will display information for that particular device, otherwise the widget will display general system info.
```
